### DISCUSSION TITLE
bagaimana cara meningkatkan memory limit dari jupyter notebook menjadi 50gb?

### DISCUSSION THREAD

**Discussion by Gustana** [teamalgoritma/community#695] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/23212456/65e7ecd8-5538-4d99-b05a-08e023222961)

saya perhatikan jupyter notebook memiliki limit untuk penggunaan memory, saya ingin meningkatkan limit tersebut menjadi 50gb.

saya telah mencari dan mememukan referensi disini
[https://stackoverflow.com/questions/57948003/how-to-increase-jupyter-notebook-memory-limit](url)
![image](https://github.com/teamalgoritma/community/assets/23212456/3173c15f-76f6-4cd9-ac0f-56e2a585ff9a)

akan tetapi saya tidak mendapatkan informasi dimanapun terkait satuan dari nilai yang dimasukkan ke max_buffer_size

**Comment by arkanayudhistira** [comment]
Dear Mas @Gustana 

Mengacu pada referensi yang anda berikan, berikut adalah beberapa hal yang perlu anda lakukan

1. Membuka Anaconda Prompt
2. Ketik `jupyter notebook --generate-config` kemudian enter
3. Anda akan diberikan directory suatu file bernama **jupyter_notebook_config.py**
4. Buka file python tersebut dari file explorer (dapat menggunakan VS Code)
5. Search `NotebookApp.max_buffer_size` menggunakan ctrl+F
6. Ganti nilai memory dengan besar memory yang diinginkan
7. Hapus tanda comment (`#`) pada bagian code tersebut
8. Save file python tersebut

Seharusnya memory limit dari jupyter dapat diatur menggunakan fungsi tersebut.

Semoga membantu!

**Reply by Gustana** [reply]
untuk satuan yang dimasukkan apa ya? apakah dalam bentuk byte, kilobyte atau apa?

**Reply by arkanayudhistira** [reply, marked as answer]
dalam bentuk Byte mas, dimana angka default nya adalah 536870912 Byte (512 MB)

**Reply by Gustana** [reply]
baik terimakasih
